misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -23,9 +23,17 @@ def __init__(
         release_mode: bool,
         aws_region: Optional[str] = None,
         namespace: str = DEFAULT_K8S_NAMESPACE,
+        materialize_url: str = "postgres://materialize:materialize@environmentd:6875/materialize",
+        materialize_internal_url: str = "postgres://materialize:materialize@environmentd:6877/materialize",
```

Comment:
I think this default is wrong, it should be mz_system without a password . Not sure how it even works right now.

Reply:
Indeed. This was already there, is it nowhere used? 😱